ui/settings.glade

```diff
@@ -4449,6 +4449,104 @@ This setting can make it easier to draw with touch. </property>
                                 <property name="position">1</property>
                               </packing>
                             </child>
+                            <child>
+                              <object class="GtkFrame">
+                                <property name="visible">True</property>
+                                <property name="can-focus">False</property>
+                                <property name="label-xalign">0</property>
+                                <child>
+                                  <object class="GtkAlignment">
+                                    <property name="visible">True</property>
+                                    <property name="can-focus">False</property>
+                                    <property name="left-padding">12</property>
+                                    <child>
+                                      <!-- n-columns=3 n-rows=3 -->
```

Comment:
Any reason why the grid has 3 columns opposed to just 1 column (in which case it could also be a GtkBox)?

Reply:
Nope, I'll fix that